vue-components/src/components/LookupMenu.vue

```diff
@@ -0,0 +1,98 @@
+<template>
+	<div :class="[ 'wikit', 'wikit-LookupMenu' ]">
+		<div
+			class="wikit-LookupMenu__item"
+			:key="index"
+			v-for="(menuItem, index) in menuItems"
+			@click="$emit( 'selected', menuItem )"
+			@mousedown.prevent
+			tabindex="0"
+		>
+			<div class="wikit-LookupMenu__item__label">
+				{{ menuItem.label }}
+			</div>
+			<div class="wikit-LookupMenu__item__description">
+				{{ menuItem.description }}
+			</div>
+		</div>
+	</div>
+</template>
+
+<script lang="ts">
+import Vue from 'vue';
+
+/**
+ * This is an internal component which used by the Lookup component.
+ */
+export default Vue.extend( {
+	name: 'LookupMenu',
+	props: {
+		menuItems: {
+			type: Array,
+			default: (): [] => [],
+		},
+	},
+	watch: {
+		menuItems(): void {
+			const menuItems = this.$el.querySelectorAll( '.wikit-LookupMenu__item' );
+			// the height automatically adjusts for up to 6 elements, then shows a scrollbar
```

Comment:
Shouldn't the comment say 7 (0 based index :P )?

Reply:
It's correct, I think!?

It checks for `if ( menuItems.length > maxNumberOfElementsDisplayed )` i.e. it does nothing special (= grows to the content height) for up to 6, and if there are more then it cuts off at `menuItems[ maxNumberOfElementsDisplayed ].offsetTop` which is the top of the 7th element. So there are 6 displayed, right?

Reply:
I also think it is correct with 6.

Reply:
looking at the storybook I see seven list items.

Reply:
Ah, you're right! Fixed, thanks :+1: